client_core/src/com/mygdx/game/client_core/network/message_handlers/GameStartedMessageHandler.java

```diff
@@ -1,30 +0,0 @@
-package com.mygdx.game.client_core.network.message_handlers;
```

Comment:
Czemu usuwasz to?

Reply:
bo dodałem to w któryś poprzednich PR w ramach 63, ale jednak miałem problem ze zrobieniem z tego implementacji o którą mi chodziło, więc usunąłem handlera i zostawiłem mniej więcej jak było kiedyś